mycroft/skills/__main__.py

```diff
@@ -199,8 +199,8 @@ def main(alive_hook=on_alive, started_hook=on_started, ready_hook=on_ready,
     # Create PID file, prevent multiple instances of this service
     mycroft.lock.Lock('skills')
     config = Configuration.get()
-    # Set the active lang to match the configured one
-    set_active_lang(config.get('lang', 'en-us'))
+    lang_code = config.get("lang", "en-us")
+    load_languages([lang_code, "en-us"])
```

Comment:
Was looking at this with Chris this morning and noticed that we were setting the default language before loading them. 
The setting was done here in the Skills main process, whilst the language loading was in the Skills Manager which is instantiated slightly further down. 
Given loading sets the default anyway, is there an issue with doing both up here?

Reply:
Not as long as this will reliably occur before other potential calls to `load_language(s)`, it should be fine.

The oopsie-sugar works both ways - if you set a default which is not loaded, it will load that language. I felt it was important to minimize the number of times Mycroft (or anybody else) would need to invoke the boilerplate.

On the other hand, the language loaders ignore languages which are already loaded. Say that five times fast. This is why the 99% of cases in which that call is `load_languages(['en-us', 'en-us'])` is essentually harmless.

Hence, it shouldn't matter which of those functions you use, nor in which order, except to skip a couple if statements.